docs/io.cozy.accounts.md

```diff
@@ -4,14 +4,19 @@
 
 The `io.cozy.accounts` doctype stores authentification informations used by [konnectors](https://github.com/konnectors) to connect to external services or API.
 
-Accounts can be managed in [Cozy-Collect](http://github.com/cozy/cozy-collect/). They are generally associated to a [`io.cozy.triggers`](io.cozy.triggers) document.
+Accounts can be managed in [Cozy-Home](http://github.com/cozy/cozy-home/) (via [Harvest](https://github.com/cozy/cozy-libs/tree/master/packages/cozy-harvest-lib)). They are generally associated to a [`io.cozy.triggers`](io.cozy.triggers) document.
 
 `io.cozy.accounts` attributes are:
 
 - `account_type` (_deprecated_) : {string} Slug of the konnector the account is related to.
 - `auth`: {object} Contains authentification data, typically a couple with `login`/`password`. It could also contain an attribute `token` for OAuth konnectors.
 - `data`: {object} Additional custom data.
 - `label`: {string} Label given by user.
+- `state`: {string} The account state is used to communicate between the konnector and [Harvest](https://github.com/cozy/cozy-libs/tree/master/packages/cozy-harvest-lib) to ask for a needed 2FA Code or to tell to reset the konnector session for example. Here are the used values for now:
+  - `TWOFA_NEEDED`: The service is asking for a Two Factor connexion and the related code (sent by the service) must be provided by the user.
+  - `TWOFA_NEEDED_RETRY`: The 2FA code provided by the user is wrong, the user can retry by providing a new one.
+  - `RESET_SESSION`: By finding this state, the konnector should reset the login session if there is one stored and reset the state.
+- `twofa_code`: When a 2FA code is asked by the service, [Harvest](https://github.com/cozy/cozy-libs/tree/master/packages/cozy-harvest-lib) will ask the user for it from and send it to the konnector via this attribute.
```

Comment:
I think all new attributes should be in camelCase by default.

Reply:
@CPatchane Any news ?

Reply:
I changed to use `twoFACode` instead